spec/requests/api/v1/users/create_spec.rb

```diff
@@ -0,0 +1,82 @@
+require 'rails_helper'
+
+describe 'User Api', type: :request do
+  let(:failed_response) { 422 }
+
+  describe 'POST api/v1/users/' do
+    let(:email)                 { 'test@test.com' }
+    let(:password)              { '12345678' }
+    let(:password_confirmation) { '12345678' }
+    let(:gender)                { 'M' }
+
+    let(:params) do
+      {
+        user: {
+          email: email,
+          password: password,
+          password_confirmation: password_confirmation,
+          gender: gender
+        }
+      }
+    end
+
+    it 'returns a successful response' do
+      post user_registration_path, params: params, as: :json
```

Comment:
Please, define this as a subject and then call it from the tests:
`subject { post user_registration_path, params: params, as: :json }`